### ISSUE TITLE
Not working if sender has sent a link and a suggestion

### ISSUE THREAD

**Issue by psamant55** [Maxhem2/instagram-mass-unsender#8]
Thanks for the script. It is does what should be the proper way. 

There is one issue and one suggestion. 

Issue: 

It doesn't move up if the sender has sent a link and that link has a preview. In my case this is the [link](https://www.myntra.com/apparel-set/inddus/inddus-women-off-white-floral-printed-regular-kurta-with-trousers-&-with-dupatta/15444940/buy)

The loops keeps running forever.

There is a suggestion. I don't know how it is but you may consider.
When we keep going up if there are hundreds of chat the UI loads and the page gets heavy. If we just delete the senders messages from the page html then the page will not be so heavy. Just a suggestion.

Thanks

**Comment by Maxhem2** [comment]
Sadly on my account I can not see link previews, [some accounts don't have the ability](https://www.reddit.com/r/Instagram/comments/sfdoyz/link_previews_dont_work/) it's an issue on Instagram's side.

If you can provide me with an account where link previews work, I'd happily fix the script.

**Comment by gl4d** [comment]
a "fix" to this is to manually delete the next message and the script will keep on going, for every link the script encounters: just manually delete the next msg and it'll "bypass" the error.

**Comment by psamant55** [comment]
@gl4d Yes and No. Yes if the link is sent by you. No If the link is from the sender.  You can't delete that.

**Comment by psamant55** [comment]
Today I checked again and it actually not issue with links only it happens if there is a reels received. 

The error is 

`ErrorUtils caught an error:

Cannot read properties of null (reading 'setAttribute')

Subsequent non-fatal errors won't be logged; see https://fburl.com/debugjs.`

**Comment by psamant55** [comment]
The small hack is to scroll up a little and they run the script again. @Maxhem2 Hope you can reproduce the issue with reel preview sent by the other person.

**Comment by gl4d** [comment]
> d

I definitely can, even if the link is sent by the other person. just tried that a min ago.
it also gets stuck at shared posts/reels, but again, the same method is working to break the loop.
